flink/interpreter/src/main/java/org/apache/zeppelin/flink/JobManager.java

```diff
@@ -110,7 +111,7 @@ public int getJobProgress(String paragraphId) {
   }
 
   public void cancelJob(InterpreterContext context) throws InterpreterException {
-    LOGGER.info("Canceling job associated of paragraph: "+ context.getParagraphId());
+    LOGGER.info("Canceling job associated of paragraph: " + context.getParagraphId());
```

Comment:
It makes sense to use `{}` placeholder instead of concatenation - it won't lead to the call of the `context.getParagraphId()` if logger is not configured for INFO

Reply:
fixed